packages/skill-api/src/core/services/process-stripe-webhook.ts

```diff
@@ -129,10 +134,22 @@ export async function processStripeWebhooks({
         }
       }
     } catch (err: any) {
-      console.error(err)
-      return {
-        status: 400,
-        body: `Webhook Error: ${err.message}`,
+      // To avoid making too much noise in Stripe, we are going to report a
+      // instance of No Associated Product to Sentry and then return a 200
+      // response from here.
+      if (err.message === NO_ASSOCIATED_PRODUCT) {
+        // Sentry.captureException(err)
```

Comment:
Do we want this to be enabled? Most of the PR is setting it up!

Reply:
just noticed that as well and already committed it back in